Applications/HexEditor/HexEditorWidget.cpp

```diff
@@ -54,11 +54,11 @@ HexEditorWidget::HexEditorWidget()
     m_editor = add<HexEditor>();
 
     m_editor->on_status_change = [this](int position, HexEditor::EditMode edit_mode, int selection_start, int selection_end) {
-        m_statusbar->set_text(0, String::format("Offset: %8X", position));
-        m_statusbar->set_text(1, String::format("Edit Mode: %s", edit_mode == HexEditor::EditMode::Hex ? "Hex" : "Text"));
-        m_statusbar->set_text(2, String::format("Selection Start: %d", selection_start));
-        m_statusbar->set_text(3, String::format("Selection End: %d", selection_end));
-        m_statusbar->set_text(4, String::format("Selected Bytes: %d", abs(selection_end - selection_start) + 1));
+        m_statusbar->set_text(0, String::formatted("Offset: {:#010X}", position));
```

Comment:
I don't know how `String::formatted` work, but `0x%08X` and `%8X` can't both be equivalent to `{:#010X}`, you must have made a mistake in at least one of these lines.

Reply:
Again, it's more explicit. (It's preference really.)